modules/javafx.controls/src/main/java/javafx/scene/control/skin/ScrollPaneSkin.java

```diff
@@ -893,7 +893,7 @@ else if (newVVal < vsb.getMin()) {
                 double vRange = getSkinnable().getVmax()-getSkinnable().getVmin();
                 double vPixelValue;
                 if (nodeHeight > 0.0) {
-                    vPixelValue = vRange / nodeHeight;
+                    vPixelValue = vRange / (nodeHeight - contentHeight);
```

Comment:
This may result in divide by 0.

Reply:
Good catch! I've also fixed a few similar issues in other places.